docs/databags.md

```diff
@@ -114,6 +114,7 @@ Docs Navigation
 [server.conf]: http://docs.splunk.com/Documentation/Splunk/latest/Admin/Serverconf
 [indexes.conf]: http://docs.splunk.com/Documentation/Splunk/latest/Admin/Indexesconf
 [user-prefs.conf]: http://docs.splunk.com/Documentation/Splunk/latest/Admin/User-prefsconf
+[limits.conf]: http://docs.splunk.com/Documentation/Splunk/latest/admin/Limitsconf
```

Comment:
This (and the change you made in the attributes doc) are just [markdown link references](http://daringfireball.net/projects/markdown/syntax#link) they don't make sense being added on their own without somewhere referring to them earlier in the doc.